src/v/cluster/ephemeral_credential_service_impl.json

```diff
@@ -0,0 +1,14 @@
+{
+    "namespace": "cluster::impl",
+    "service_name": "ephemeral_credential",
+    "includes": [
+        "cluster/ephemeral_credential_serde.h"
+    ],
+    "methods": [
+        {
+            "name": "put_ephemeral_credential",
+            "input_type": "put_ephemeral_credential_request",
+            "output_type": "put_ephemeral_credential_reply"
+        }
+    ]
+}
```

Comment:
nit: newline

Or do we not do that for json files?

Reply:
Done